2/contrib/s2i/run

```diff
@@ -281,6 +354,8 @@ else
   fi  
 fi
 
+echo ${OPENSHIFT_JENKINS_IMAGE_VERSION} > ${JENKINS_HOME}/image-version
```

Comment:
presumably could do this up where you're checking for this value.  (if it's not there/doesn't match, do this)

Reply:
bump

Reply:
(just trying to get all the version marking logic as close together as possible)

Reply:
sorry missed that one ... agreed, will push momentarily